kasa/cli.py

```diff
@@ -248,7 +269,10 @@ async def print_discovered(dev: SmartDevice):
             echo()
 
     await Discover.discover(
-        target=target, timeout=timeout, on_discovered=print_discovered
+        target=target,
+        timeout=timeout,
+        on_discovered=print_discovered,
+        on_unsupported=on_unsupported,
     )
 
```

Comment:
Please add here a print out summary of detected devices, that will help users to discover the `--show-unsupported`. Something like this:
```

    click.echo(f"Found {len(discovered)} devices")
    if unsupported:
        click.echo(f"Found {len(unsupported)} unsupported devices, use --show-unsupported to show them")
```

Reply:
Done